docs/dictionary/en-custom.txt

```diff
@@ -204,6 +205,7 @@ makefiles
 manpage
 mawxlihjizaogicbjyxbzig
 mawxlihjizcbwb
+mergeby
```

Comment:
not sure this one is needed? I don't match it in the doc, only in .yml... ?

Reply:
Thank you. I may have modified observing some other failure.